### PR TITLE
[toranj] improve code coverage by using absolute path

### PR DESCRIPTION
**Pull request by bukepo**
Gcov sometimes cannot correctly find source files when the code is
configured with relative path. This PR enables using absolute path to
run toranj tests.

### CODE REVIEW

File: tests/toranj/build.sh [hunk 1]
```diff
@@ -91,13 +91,22 @@ configure_options="                \
...
+
 case ${build_config} in
     ncp)
         echo "==================================================================================================="
         echo "Building OpenThread NCP FTD mode with POSIX platform"
         echo "==================================================================================================="
         ./bootstrap || die
-        ./configure                             \
+        cd "${top_builddir}"
+        ${top_srcdir}/configure              \
```
Comment (abtink) [inline tests/toranj/build.sh]: nit: align `\` with other lines?

File: tests/toranj/build.sh [hunk 2]
```diff
@@ -109,7 +118,8 @@ case ${build_config} in
         echo "Building OpenThread RCP (NCP in radio mode) with POSIX platform"
         echo "===================================================================================================="
         ./bootstrap || die
-        ./configure                             \
+        cd "${top_builddir}"
+        ${top_srcdir}/configure              \
```
Comment (abtink) [inline tests/toranj/build.sh]: nit: align `\` with other lines?

File: tests/toranj/build.sh [hunk 3]
```diff
@@ -125,7 +135,8 @@ case ${build_config} in
         echo "Building OpenThread POSIX App NCP"
         echo "===================================================================================================="
         ./bootstrap || die
-        ./configure                             \
+        cd "${top_builddir}"
+        ${top_srcdir}/configure              \
```
Comment (abtink) [inline tests/toranj/build.sh]: nit: align `\` with other lines?

**Comment by bukepo** [comment]
Thanks for the help. @jwhui

**Pull request merged**